src/ui/macstr_ui.py

```diff
@@ -21,7 +22,7 @@ def setupUi(self, MainWindow):
 
         # loading in the JSON data
         with open("../../notebooks/test_image_coords.json", "r") as contents:
-            image_data = yaml.safe_load(contents)
+            image_data = yaml.safe_load(contents)  # noqa: F841
```

Comment:
should it be ignored? Why load if not used? Maybe not used yet?

Reply:
For now it should be ignored. But once I start loading in the `img-coords` into the GUI, the `image_data` variable will be used. 

It will be used once I submit the GUI PR.